### PR TITLE
LibJS-Bytecode: Add a highlighter for LibJS' bytecode

### PR DESCRIPTION
**Pull request by Hendiadyoin1**
As always things left to do are updating the Readme,
for that:

How to obtain:
```sh
Build/lagom/js -d $file 2>&1 | code - 
```
Note: This usually includes the colouring escape codes

Some sections have less than ideal tokens, like the headers are `headers` which have surprisingly bad visibility in the default `Dark+` theme, and nearly everything is a `string`

### CODE REVIEW

File: syntaxes/serenity-libjs-bytecode.tmLanguage.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+						"1": { "name": "constant.numeric.libjs.bytecode" },
+						"2": { "name": "punctuation.colon.libjs.bytecode" },
+						"3": { "name": "string.libjs.bytecode" }
+					}
+				}
+			],
+			"end": "^$"
+		}
+	}
+}
```
Comment (kleinesfilmroellchen) [inline syntaxes/serenity-libjs-bytecode.tmLanguage.json:162]: Add a newline

File: syntaxes/serenity-libjs-bytecode.tmLanguage.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
+{
+	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+	"scopeName": "source.serenity-libjs-bytecode",
```
Comment (kleinesfilmroellchen) [inline syntaxes/serenity-libjs-bytecode.tmLanguage.json:3]: This has to be the same as the scopes used below. `source.libjs.bytecode`

File: package.json [hunk 3]
```diff
@@ -76,6 +84,11 @@
                 "language": "web-idl-serenity",
                 "scopeName": "source.serenity-idl",
                 "path": "./syntaxes/serenity-web-idl.tmLanguage.json"
+            },
+            {
+                "language": "libjs-bytecode-serenity",
+                "scopeName": "source.serenity-libjs-bytecode",
```
Comment (kleinesfilmroellchen) [inline package.json:90]: Same problem

File: README.md [hunk 4]
```diff
@@ -40,6 +40,11 @@ The GML formatter needs to save the file in order to format it, so formatting an
 
 - Extended attribute highlighting may not be spec-compliant.
 
+### LibJS-Bytecode
```
Comment (kleinesfilmroellchen) [inline README.md:43]: That doesn't have to have a dash.

**Review by kleinesfilmroellchen — CHANGES_REQUESTED** [review]

**Review by kleinesfilmroellchen — CHANGES_REQUESTED** [review]
image

There appears to be a problem with highlighting braces around quoted text.

**Comment by Hendiadyoin1** [comment]
Yeah, I did not expect that spaces might appear in an argument, so I made it allow spaces in Strings

**Pull request merged**